src/expr/term_context.cpp

```diff
@@ -131,7 +131,7 @@ uint32_t PolarityTermContext::getValue(bool hasPol, bool pol)
 
 void PolarityTermContext::getFlags(uint32_t val, bool& hasPol, bool& pol)
 {
-  hasPol = val == 0;
+  hasPol = val != 0;
```

Comment:
What consequences does this change have?

Reply:
This corrects the value of `hasPol` for this method, RelevantDomain in quantifiers was the only current user of this class, which was used as a heuristic for enumerative instantiation. In summary, this means that enumerative instantiation (might) be slightly better after this commit. (Regardless of whether it is, the code should be corrected).